package.json

```diff
@@ -16,14 +16,20 @@
         "worker_threads": "./externals/worker_threads.js"
     },
     "dependencies": {
+        "@tinymce/tinymce-react": "^4.1.0",
+        "@types/draft-convert": "^2.1.4",
+        "@types/draft-js": "^0.11.9",
         "@types/react": "^17.0.15",
         "@types/react-dom": "^17.0.9",
+        "@types/react-draft-wysiwyg": "^1.13.4",
         "@unirep/circuits": "git+https://github.com/Unirep/circuits.git#alpha",
         "@unirep/crypto": "git+https://github.com/Unirep/crypto.git",
         "@unirep/unirep": "git+https://github.com/Unirep/unirep.git#alpha",
         "@unirep/unirep-social": "git+https://github.com/Unirep/unirep-social.git#alpha",
         "bootstrap": "^5.0.2",
         "dateformat": "^4.5.1",
+        "draft-convert": "^2.1.12",
+        "draft-js": "^0.11.7",
```

Comment:
I'm really not a fan of this package. I've only used the online demo but the editor seems complicated. I like markdown because I can write it without needing buttons and a full editor to make it work. Also the minified size of this package is 164KB.

The biggest problem with this editor is we don't know what the post will look like. The editor shows one thing but once it's posted it will be rendered using `markdown-it` which will make the post look different. 

I think a good first draft of the markdown editor is a two button selector with the option `edit` and `preview`. If `edit` is selected a text field should be shown below. If `preview` is selected the text field should go away and the text should be rendered to HTML and shown.

We can build this with no new packages and it will show the user exactly what they will be posting.

Reply:
ok, then I'll rewrite this part by using only `markdown-it`.
just give the new package a try, but I also found many bugs of it, also fixing it for a lot of time.